cg-spec/editors_draft.html

```diff
@@ -626,6 +641,20 @@ <h2>Triples and occurrences</h2>
         -->
       </pre>
 
+      <p>The same kind of pattern can be used in the classical example of Elisabeth Taylor's two marriages with Richard Burton. If their marrying is represented by a triple `:lizTaylor :married :richardBurton` (which, by definition, can not occur multiple times in a graph), then additional nodes must be introduced to represent the different occurrences.
+
+        <pre data-transform="updateExample" id="married-example"
+        data-content-type="text/turtle"
+        class="nohighlight example"
+      >
+        <!--
+        :lizTaylor :married :richardBurton {| :occurrence
+            [ :year 1964; :location :montreal ],
+            [ :year 1975, :location :botswana ]
+        |}.
+        -->
+      </pre>
+
```

Comment:
I am not sure whether including this example here is a good idea. It can be misleading for two reasons: First, the `:occurrence` property used here has a completely different meaning than the inverse of the `:occurrenceOf` property used in the previous example; in particular, the `:occurrence` property here is not about the occurrence of triples (which is the topic of this section) but about the occasion of a marriage event. Second, this example assumes referential transparency (i.e., the `:occurrence` property should be a TEP).

Reply:
I agree that the similarity in names may be misleading (this was also raised by Thomas on the mailing list), and should be fixed. Maybe `':event` would be a better choice of name ?

I would like to keep the example, though, precisely because it demonstrates that the type-token distinction shows up in different use-cases:
- unasserted or asserted triples
- with "opaque occurrences" or with "transparent occurrences" (because yes, `:occurrence`/`:event` in this example would be a TEP)